Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeHost.mm

```diff
@@ -198,7 +198,7 @@
     }
 
 #if ENABLE(THREADED_ANIMATION_RESOLUTION)
-    Ref { *m_drawingArea }->registerTimelineIfNecessary(processIdentifier, transaction.acceleratedTimelineTimeOrigin(), MonotonicTime::now());
+    Ref { *m_drawingArea }->updateTimelineRegistration(processIdentifier, transaction.timelines(), MonotonicTime::now());
```

Comment:
A single process can send multiple transactions (with site isolation), which would then overwrite each other in the registry. Please add a TODO/bug for fixing this!

Reply:
This is tracked by [bug 301261](https://bugs.webkit.org/show_bug.cgi?id=301261).